static/test/javascripts-legacy/spec/facia/weather.spec.js

```diff
@@ -16,7 +16,7 @@ define([
         beforeEach(function (done) {
 
             injector.require(['facia/modules/onwards/weather', 'lib/config'], function () {
-                sut = arguments[0];
+                sut = arguments[0].Weather;
```

Comment:
Could you please add a comment `#? move to jest as sonn ...` at the top of the file, so we don't forget about the reason?